src/connection.toit

```diff
@@ -300,11 +299,11 @@ class ContentLengthWriter implements BodyWriter:
   is_done -> bool:
     return remaining_length_ == 0
 
-  write data:
+  write data -> none:
```

Comment:
I'd rather fix this one to return the size, instead of fixing the other to not do so.

Reply:
Agree.